### PR TITLE
optimize: classify script to correspond directory

### PR DESCRIPTION
**Pull request by helloworlde**
### Ⅰ. Describe what this PR did

1. Classify script to correspond directory

### CODE REVIEW

File: script/client/at/db/mysql.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+(
+    `id`            BIGINT(20)   NOT NULL AUTO_INCREMENT COMMENT 'increment id',
+    `branch_id`     BIGINT(20)   NOT NULL COMMENT 'branch transaction id',
+    `xid`           VARCHAR(100) NOT NULL COMMENT 'global transaction id',
+    `context`       VARCHAR(128) NOT NULL COMMENT 'undo_log context,such as serialization',
+    `rollback_info` LONGBLOB     NOT NULL COMMENT 'rollback info',
+    `log_status`    INT(11)      NOT NULL COMMENT '0:normal status,1:defense status',
+    `log_created`   DATETIME     NOT NULL COMMENT 'create datetime',
+    `log_modified`  DATETIME     NOT NULL COMMENT 'modify datetime',
+    `ext`           VARCHAR(100) DEFAULT NULL COMMENT 'reserved field',
```
Comment (zjinlei) [inline script/client/at/db/mysql.sql]: ext has removed
oracle,pgsql too

File: script/client/at/db/oracle.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    PRIMARY KEY (id),
+    CONSTRAINT ux_undo_log UNIQUE (xid, branch_id)
+);
+
+COMMENT ON TABLE undo_log IS 'AT transaction mode undo table';
+
+-- Generate ID using sequence and trigger
+CREATE SEQUENCE undo_log_seq START WITH 1 INCREMENT BY 1;
+
+CREATE OR REPLACE TRIGGER undo_log_seq_tr
```
Comment (jsbxyyx) [inline script/client/at/db/oracle.sql]: code implement specify undo_log_seq.nextval. here need auto generate?

Comment (helloworlde) [reply script/client/at/db/oracle.sql]: I'm not sure, but MySQL and PG use auto_increment, does there has different in these?

Comment (slievrly) [reply script/client/at/db/oracle.sql]: need `undo_log_seq`

Comment (helloworlde) [reply script/client/at/db/oracle.sql]: Had edited.

**Comment by zjinlei** [comment]
Add readme under seata-server resources to guide users to script

**Comment by zjinlei** [comment]
Update the saga table creation statement synchronously.
refer: https://github.com/seata/seata/pull/1975

File: script/client/saga/db/postgresql.sql [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    parent_id           VARCHAR(46),
+    gmt_started         TIMESTAMP(0)               NOT NULL,
+    business_key        VARCHAR(48),
+    start_params        TEXT,
+    gmt_end             TIMESTAMP(0) DEFAULT now(),
+    excep BYTEA,
+    end_params          TEXT,
+    status              VARCHAR(2),
+    compensation_status VARCHAR(2),
+    is_running          SMALLINT,
```
Comment (zjinlei) [inline script/client/saga/db/postgresql.sql]: test case failed: column "is_running" is of type smallint but expression is of type boolean

File: server/src/main/resources/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- db: Create table script for server when store mode is `db`
+
+## config-center
+
+> Store initialize script for configuration center, will use `config.txt` as configuration when initial
+
+- nacos: Initialize script for Nacos
+- zk: Initialize script for ZooKeeper, the script need related script in Zookeeper, you need download yourself. You can modify `zk-params.txt` to change the ZooKeeper server configuration, or input when execute also
+- apollo: Initialize script for Apollo. You can modify `apollo-params.txt` to change the Apollo server configuration, or input when execute also
+- etcd3: Initialize script for Etcd
```
Comment (xingfudeshi) [inline server/src/main/resources/README.md]: Initialize script for Etcd3

File: server/src/main/resources/README-zh.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- db: server 侧的保存模式为 `db` 时所需表的建表语句
+
+## config-center
+
+> 用于存放各种配置中心的初始化脚本，执行时都会读取 `config.txt`配置文件，并写入配置中心
+
+- nacos: 用于向 Nacos 中添加配置
+- zk: 用于向 Zookeeper 中添加配置，脚本依赖 Zookeeper 的相关脚本，需要手动下载；ZooKeeper相关的配置可以写在 `zk-params.txt` 中，也可以在执行的时候输入
+- apollo: 向 Apollo 中添加配置，Apollo 的地址端口等可以写在 `apollo-params.txt`，也可以在执行的时候输入
+- etcd3: 用于向 Etcd 中添加配置
```
Comment (xingfudeshi) [inline server/src/main/resources/README-zh.md]: Etcd3

File: script/server/db/oracle.sql [hunk 6]
```diff
@@ -0,0 +1,56 @@
+-- -------------------------------- The script used when storeMode is 'db' --------------------------------
```
Comment (jsbxyyx) [inline script/server/db/oracle.sql]: oracle field or index add comment?

**Comment by jsbxyyx** [comment]
oracle field or index add comment?
postgresql field or index add comment?

**Comment by helloworlde** [comment]
> oracle field or index add comment?
> postgresql field or index add comment?

I will add comments later

**Pull request merged**